services/nutanix-ai/metadata.yaml

```diff
@@ -134,12 +135,17 @@ overview: |-
   ### After enabling Nutanix Enterprise AI, Update the endpoint of istio-ingressgateway service external IP or FQDN in the Nutanix AI dashboard ConfigMap.
   On the terminal, run the following command to get the external IP or FQDN of the istio-ingressgateway service from respective cluster:
   ```
-  export NAI_UI_ENDPOINT=https://$(kubectl get svc istio-ingressgateway -n istio-system -ojsonpath='{.status.loadBalancer.ingress[0].ip}')/
+  export NAI_UI_ENDPOINT=https://$(kubectl get svc istio-ingressgateway -n istio-system -o json | jq -r '.status.loadBalancer.ingress[0] | (.hostname // .ip)')/
   ```
   Update the dashboard URL in config map as follows:
   ```
   kubectl patch cm nai-ui -n <workspace_namespace> -p '{"data":{"dashboardLink":"'${NAI_UI_ENDPOINT}'"}}'
   ```
+  Run below patches:
+  ```
+  kubectl patch configmap config-features -n knative-serving --patch '{"data":{"kubernetes.podspec-nodeselector":"enabled"}}'
```

Comment:
are these two CMs override CM that are provided in one our knative chart/knative kommander app?

Reply:
those are related to solve the issue mentioned in https://jira.nutanix.com/browse/NCN-104322. We are planning to put those to manual steps in 2.13 release.